src/Wellcome.Dds/Wellcome.Dds.Dashboard/Controllers/DashController.cs

```diff
@@ -0,0 +1,840 @@
+        private readonly ISimpleCache cache;
+        private readonly IStatusProvider statusProvider;
+        private readonly IDatedIdentifierProvider recentlyDigitisedIdentifierProvider;
+        private readonly IWorkStorageFactory workStorageFactory;
+        private readonly CacheBuster cacheBuster;
+        private readonly DdsOptions ddsOptions;
+        private readonly DlcsOptions dlcsOptions;
+        private readonly IDds dds;
+        private readonly IWebHostEnvironment webHostEnvironment;
+        private readonly IWorkflowCallRepository workflowCallRepository;
+
+        const string CacheKeyPrefix = "dashcontroller_";
+        const int CacheSeconds = 5;
+        private Synchroniser synchroniser;
+
+        public DashController(
+            IDashboardRepository dashboardRepository,
+            ISimpleCache cache,
+            IIngestJobRegistry jobRegistry,
+            IIngestJobProcessor jobProcessor,
+            IStatusProvider statusProvider,
+            Synchroniser synchroniser,
+            IDatedIdentifierProvider recentlyDigitisedIdentifierProvider,
+            IWorkStorageFactory workStorageFactory,
+            CacheBuster cacheBuster,
+            IOptions<DdsOptions> ddsOptions,
+            IOptions<DlcsOptions> dlcsOptions,
+            IDds dds,
+            IWebHostEnvironment webHostEnvironment,
+            IWorkflowCallRepository workflowCallRepository)
+        {
+            this.dashboardRepository = dashboardRepository;
+            this.cache = cache;
+            this.jobRegistry = jobRegistry;
+            this.jobProcessor = jobProcessor;
+            this.statusProvider = statusProvider;
+            this.synchroniser = synchroniser;
+            this.recentlyDigitisedIdentifierProvider = recentlyDigitisedIdentifierProvider;
+            this.workStorageFactory = workStorageFactory;
+            this.cacheBuster = cacheBuster;
+            this.ddsOptions = ddsOptions.Value;
+            this.dlcsOptions = dlcsOptions.Value;
+            this.dds = dds;
+            this.webHostEnvironment = webHostEnvironment;
+            this.workflowCallRepository = workflowCallRepository;
+            //this.cachingPackageProvider = cachingPackageProvider;
+            //this.cachingAltoSearchTextProvider = cachingAltoSearchTextProvider;
+            //this.cachingAllAnnotationProvider = cachingAllAnnotationProvider;
+        }
+
+        public ActionResult Index()
+        {
+            var recent = recentlyDigitisedIdentifierProvider.GetDatedIdentifiers(100);
+            return View(recent);
+        }
+
+        public ActionResult Status(int page = 1)
+        {
+            var problemJobs = jobRegistry.GetProblems(statusProvider.EarliestJobToTake);
+            Page<ErrorByMetadata> errorsByMetadataPage;
+            try
+            {
+                errorsByMetadataPage = dashboardRepository.GetErrorsByMetadata(page);
+            }
+            catch (Exception)
```

Comment:
Worth catching and logging exception?

Reply:
Oddly considering the dependency list comment above - this controller doesn't yet have a logger!
Going to leave for now